tools/check_api_approvals.sh

```diff
@@ -142,6 +142,16 @@ if [ "${NEW_OP_ADDED}" != "" ] && [ "${GIT_PR_ID}" != "" ]; then
     fi
 fi
 
+OP_FILE_CHANGED=`git diff --name-only --diff-filter=AM upstream/$BRANCH |grep -oE ".+_op..*" || true`
+if [ "${OP_FILE_CHANGED}" != "" ] && [ "${GIT_PR_ID}" != "" ]; then
+    CHECK_SHAREDATAWITH=`git diff -U0 --diff-filter=AM upstream/$BRANCH |grep "+" |grep -oE "ShareDataWith[(]" || true`
+    CHECK_SHAREBUFFERWITH=`git diff -U0 --diff-filter=AM upstream/$BRANCH |grep "+" |grep -oE "ShareBufferWith[(]" || true`
+    if [ "${CHECK_SHAREDATAWITH}" != "" ] || [ "${CHECK_SHAREBUFFERWITH}" != "" ]; then
+        echo_line="If you use the ShareDataWith or ShareBufferWith method, please change the method to others. Because this method cloud casus some bug of memory. If you don't want to change the method, you must have one RD (lanxianghit or luotao1 or sneaxiy) approval for the usage of other method.\n"
```

Comment:
Re-write these English sentences ...

"Using ShareDataWith or ShareBufferWith is not recommended. You must have one RD's (lanxianghit or luotao1 or sneaxiy) approval to use these methods"

Reply:
sneaxiy(Recommend) or luotao1 or lanxianghit

Reply:
Done.